java/openjdk8/Portfile

```
@@ -21,13 +21,15 @@ description         OpenJDK 8
 long_description    JDK 8 and JRE 8 builds of OpenJDK, the Open-Source implementation \
                     of the Java Platform, Standard Edition, and related projects.
 homepage            https://openjdk.org/
-master_sites        https://git.openjdk.org/jdk8u/archive/refs/tags/
+master_sites        https://git.openjdk.org/jdk8u/archive/refs/tags/ \
+                    https://github.com/openjdk/jdk8u/archive/refs/tags/
```

https://openjdk.org/projects/jdk/ shows that the official repository is now on GitHub:

```suggestion
master_sites        https://github.com/openjdk/jdk8u/archive/refs/tags/
```